recipes/libdispatch/all/conanfile.py

```diff
@@ -21,9 +21,6 @@ class LibDispatchConan(ConanFile):
     _cmake = None
 
     def validate(self):
-        if self.settings.build_type != "Release":
-            raise ConanInvalidConfiguration("Just Release builds allowed.")
-
         if self.settings.compiler != "clang":
```

Comment:
what about apple-clang?

Reply:
It is not necessary to build for apple-clang becase libdispatch is distributed with macOS.
Of course you can do it, but it is worthless.

Reply:
So you should raise for Macos, prefer to be explicit.

Reply:
The thing is that GCC is not allowed, also.

Reply:
And there are no instructions to build on Windows.

Reply:
```suggestion
        if self.settings.compiler != "clang" and self.settings.os != "Linux":
```